website/blog/2021-05-17-codepush.md

```diff
@@ -0,0 +1,409 @@
+Thanks to CodePush, you are able to deliver a new bundle (which might contain updated JavaScript files, new images, etc.) without having to fully redeploy a new package. This means, for example, that you can update the content of your application published on a Store without submitting a new package and waiting for the certification process to be completed; or that you can add a new feature in your LOB app without having to redeploy it via Intune or another MDM solution to all your employees.
+The only caveat is that **the feature works only for updates which involves the bundle**, which means changes in the JavaScript code, a new asset, etc. If the update involves changes in the binary package (like adding a new capability in the manifest or integrating a new native module), then you are required to submit a new package using the standard approach. 
+
+Enabling Code Push requires to interact with three different layers:
+
+1. App Center, which is the cloud platform that will host and deliver your updates. You're going to push new bundles to App Center using the [App Center CLI](https://docs.microsoft.com/en-us/appcenter/cli/).
+2. A native module, which must be integrated in the React Native application.
+3. A set of JavaScript APIs, which you must integrate in the JavaScript layer of your application to manage the update lifecycle.
+
+CodePush initially supported only iOS and Android, but [the native module](https://github.com/microsoft/react-native-code-push/) has recently added support for the Windows implementation of React Native as well. However, App Center still lacks support for this technology on Windows and, today, when you create a new Windows application on App Center, React Native isn't one of the available options. This means that the infrastructure needed to support CodePush won't be available. In this article we're going to see how we can configure our Windows application to use Code Push, even if it isn't fully supported by App Center.
+
+## Setting up App Center
+
+The first task is to create a new application in Visual Studio App Center. Thes starting point is the [App Center website](https://appcenter.ms/), where you can login to your dashboard. If you don't already have an account, you can create a free one the first time you login using your Microsoft Account, work account, GitHub, Facebook or Google account.
+Once you are logged in, click on **Add new** and choose **Add new app**.
+
+This is the first step where we have to apply a workaround. In this panel, you will be able to setup your application, by specifing the name and the target platform:
+
+![](assets/2021-05-17-codepush/NewApp.png)
+
+However, as previously mentioned, if you choose **Windows** you won't see React Native as an option. As such, we have to trick App Center to think that our Windows application is actually an Android one.
```

Comment:
Is this is a point-in-time thing / do we have a line of sight to fixing this?

Reply:
It's actually both. Yes, the post is meant to be a "point-in-time" snapshot to unlock who wants to adopt CodePush today. At the same time, App Center is currently in "slow down mode", so I'm not sure if and when Windows support for React Native might be picked up.